tests/sphere_test.cpp

```diff
@@ -0,0 +1,125 @@
+#include "obj/objects.h"
+#include "gtest/gtest.h"
+
+#include <random>
+
+using core::Ray;
+using core::RayHitData;
+using core::Vec3;
+using objects::Sphere;
+
+const float kSkipFreq = 1000;
+const Vec3 kVecZero(0, 0, 0);
+
+class RandomFloatGenerator {
+public:
+  RandomFloatGenerator(float min, float max)
```

Comment:
1. How is this min and max for a normal distribution? It looks like you are generating numbers around -1 with stddev of 1 which is likely not what you want.
2. In tests, avoid having non-deterministic random seed. Please fix the seed.